src/xcode/ENA/ENA/Source/AppDelegate & Globals/NotificationManager.swift

```diff
@@ -91,18 +91,16 @@ final class NotificationManager: NSObject, UNUserNotificationCenterDelegate {
 	// MARK: - Internal
 	
 	// Internal for testing
-	func extract(_ prefix: String, from: String) -> (HealthCertifiedPerson, HealthCertificate)? {
-		guard from.hasPrefix(prefix) else {
```

Comment:
Why is this not needed anymore?

Reply:
we check the prefix inside the checkForLocalNotificationsActions function as we need to know the enum of the notification for the switch cases